### PR TITLE
AGMS-48 remove keycloak mentions

### PR DESCRIPTION
**Pull request by finp**
use tags and attributes to make IDM service docs work for keycloak and RHSSO

### CODE REVIEW

File: modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc [hunk 1]
```diff
@@ -95,6 +106,7 @@ Depending on the target platform, set the additional value as described in eithe
 
 . Save your changes.
 
-. Create a new user account as described in link:https://www.keycloak.org/docs/3.3/server_admin/topics/users/create-user.html[Creating a New User].
+. Creating a new user account as described in link:{docs-create-user}[Creating a New User].
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc]: @finp - should this follow this syntax? link:url[text^]

File: modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc [hunk 2]
```diff
@@ -95,6 +106,7 @@ Depending on the target platform, set the additional value as described in eithe
...
+
+. Set up credentials for the new user as described in link:{docs-user-creds}[User Credentials].
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc]: @finp - should this follow this syntax? link:url[text^]

File: modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc (hunk 1 above)
Comment (finp) [reply modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc]: yes, but given the already large scope of this Pr, lets handle it separately

**Comment by finp** [comment]
@jstaffor can you take a look, I think I've addressed https://github.com/aerogear/mobile-docs/pull/612 here too, so feel free to close that to avoid a rebase

**Comment by jstaffor** [comment]
Closed https://github.com/aerogear/mobile-docs/pull/612 in favor of this :+1:

**Pull request merged**